test/generators/TraderDetailsConsignmentAnswersGenerator.scala

```diff
@@ -16,53 +16,72 @@
 
 package generators
 
-import models.UserAnswers
+import models.{SecurityDetailsType, UserAnswers}
 import org.scalacheck.{Arbitrary, Gen}
+import pages.preTaskList.SecurityDetailsTypePage
 import pages.traderDetails.consignment._
-import play.api.libs.json.JsBoolean
+import play.api.libs.json.{JsBoolean, JsString}
 
 trait TraderDetailsConsignmentAnswersGenerator extends UserAnswersGenerator {
   self: Generators =>
 
-  lazy val arbitraryTraderDetailsConsignmentAnswersWithConsignor: Gen[UserAnswers] = arbitraryUserAnswers(
-    arbitraryTraderdetailsConsignmentApprovedOperatorUserAnswersEntry.arbitrary ::
+  lazy val arbitraryTraderDetailsConsignmentAnswersWithConsignorAndConsignee: Gen[UserAnswers] = arbitraryUserAnswers(
+    Arbitrary((SecurityDetailsTypePage, JsString(SecurityDetailsType.EntrySummaryDeclarationSecurityDetails.toString))).arbitrary ::
```

Comment:
I believe this can be `Json.toJson[SecurityDetailsType](SecurityDetailsType.EntrySummaryDeclarationSecurityDetails)` if you prefer